xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/accounts/NfTokenObject.java

```diff
@@ -0,0 +1,43 @@
+package org.xrpl.xrpl4j.model.client.accounts;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
+import org.xrpl.xrpl4j.model.transactions.NfTokenId;
+
+import java.util.Optional;
+
+/**
+ * Structure of an NFToken stored on the ledger.
+ */
+@Value.Immutable
+@JsonSerialize(as = ImmutableNfTokenObject.class)
+@JsonDeserialize(as = ImmutableNfTokenObject.class)
+public interface NfTokenObject {
+
+  /**
+   * Construct a builder for this class.
+   *
+   * @return An {@link ImmutableNfTokenObject.Builder}.
+   */
+  static ImmutableNfTokenObject.Builder builder() {
+    return ImmutableNfTokenObject.builder();
+  }
+
+  /**
+   * The unique TokenID of the token.
+   *
+   * @return The unique TokenID of the token.
+   */
+  @JsonProperty("NFTokenID")
+  NfTokenId tokenId();
```

Comment:
does it make sense to call this `nftokenid`? and perhaps alias that as `id`?

Reply:
nfTokenId is more descriptive and makes more sense. I change it to that

Reply:
the method itself should be called `nfTokenID()` instead of `tokenId()` too, right?